src/vivarium_public_health/dataset_manager.py

```diff
@@ -112,11 +112,11 @@ def _uncached_load(self, entity_key, keep_age_group_edges, column_filters):
         if entity_key.to_path() not in self._hdf:
             raise ArtifactException(f"{entity_key.to_path()} should be in {self.artifact_path}")
 
-        node = self._hdf._handle.get_node(entity_key.to_path())
+        node = self._hdf.get_node(entity_key.to_path())
 
         if "NODE_TYPE" in dir(node._v_attrs) and node.get_attr("NODE_TYPE") == "file":
```

Comment:
I'm pretty sure we can use a different mechanism for this same thing without accessing private attributes.  I.e. a quick inspection of one of the artifact hdf files shows that the json blobs are always stored as `tables.Leaf` instances while the data frames are stored as groups (since they are collections of arrays).

Reply:
Interesting. I was never actually able to find the json in the example artifact so I couldn't play with it. Sounds like this can be reworked

Reply:
If you add any risk factor, its distribution or restrictions comes in as json